src/test/rgw/rgw_multi/tests_ps.py

```diff
@@ -493,82 +294,6 @@ def stop_kafka_receiver(receiver, task):
 # (2) our client is not using the keystore, and the "rootCA.crt" file generated in the process above
 # should be copied to: $KAFKA_DIR
 
-def init_kafka():
-    """ start kafka/zookeeper """
-    try:
-        KAFKA_DIR = os.environ['KAFKA_DIR']
-    except:
-        KAFKA_DIR = ''
-
-    if KAFKA_DIR == '':
-        log.info('KAFKA_DIR must be set to where kafka is installed')
-        print('KAFKA_DIR must be set to where kafka is installed')
-        return None, None, None
-    
-    DEVNULL = open(os.devnull, 'wb')
-
-    print('\nStarting zookeeper...')
-    try:
-        zk_proc = subprocess.Popen([KAFKA_DIR+'bin/zookeeper-server-start.sh', KAFKA_DIR+'config/zookeeper.properties'], stdout=DEVNULL)
-    except Exception as error:
-        log.info('failed to execute zookeeper: %s', str(error))
-        print('failed to execute zookeeper: %s' % str(error))
-        return None, None, None
-
-    time.sleep(5)
-    if zk_proc.poll() is not None:
-        print('zookeeper failed to start')
-        return None, None, None
-    print('Zookeeper started')
-    print('Starting kafka...')
-    kafka_log = open('./kafka.log', 'w')
-    try:
-        kafka_env = os.environ.copy()
-        kafka_env['KAFKA_OPTS']='-Djava.security.auth.login.config='+KAFKA_DIR+'config/kafka_server_jaas.conf'
-        kafka_proc = subprocess.Popen([
-            KAFKA_DIR+'bin/kafka-server-start.sh', 
-            KAFKA_DIR+'config/server.properties'], 
-            stdout=kafka_log,
-            env=kafka_env)
-    except Exception as error:
-        log.info('failed to execute kafka: %s', str(error))
-        print('failed to execute kafka: %s' % str(error))
-        zk_proc.terminate()
-        kafka_log.close()
-        return None, None, None
-
-    # TODO add kafka checkpoint instead of sleep
-    time.sleep(15)
-    if kafka_proc.poll() is not None:
-        zk_proc.terminate()
-        print('kafka failed to start. details in: ./kafka.log')
-        kafka_log.close()
-        return None, None, None
-
-    print('Kafka started')
-    return kafka_proc, zk_proc, kafka_log
-
-
-def clean_kafka(kafka_proc, zk_proc, kafka_log):
-    """ stop kafka/zookeeper """
-    try:
-        kafka_log.close()
-        print('Shutdown Kafka...')
-        kafka_proc.terminate()
-        time.sleep(5)
-        if kafka_proc.poll() is None:
-            print('Failed to shutdown Kafka... killing')
-            kafka_proc.kill()
-        print('Shutdown zookeeper...')
-        zk_proc.terminate()
-        time.sleep(5)
-        if zk_proc.poll() is None:
-            print('Failed to shutdown zookeeper... killing')
-            zk_proc.kill()
-    except:
-        log.info('kafka/zookeeper already terminated')
-
-
 def init_env(require_ps=True):
```

Comment:
you can remove the `require_ps` parameter - now this file has only pubsub tests

Reply:
Okay.

Reply:
you did not remove it, it is still there

Reply:
I'll remove this.